src/main/scala/com/zeotap/data/io/source/beam/interpreters/BeamInterpreters.scala

```diff
@@ -26,6 +26,7 @@ object BeamInterpreters {
         case Driver(driver) => pCollectionReader.option("driver", driver)
         case TableName(tableName) => pCollectionReader.option("query", f"select * from $tableName")
         case Query(query) => pCollectionReader.option("query", query)
+        case DistributedLoad(_, _, _) => throw new IllegalArgumentException("This Feature is not yet supported in Beam!")
```

Comment:
I think this can be removed as the feature is not added to the BeamLoader class anyway

Reply:
let's leave it there as a fail-safe just in case